README.md

```diff
@@ -1,12 +1,13 @@
 
-AliceO2
-=======
+ALICE O2 software 
```

Comment:
there is an extra blank at the end, but we do not have a tool the check for formatting guideline "Never put trailing whitespace at the end of a line"